code/software/libs/src/cstdlib/include/ctype.h

```diff
@@ -18,7 +18,7 @@
 
 #include <stdint.h>
 
-typedef   uint32_t    size_t;
+typedef unsigned long size_t;
```

Comment:
Yes, I agree for that file.  I think I accidentally left that in there double checking that types were all matching - hard to know what gcc "really" uses since hidden in __INT_TYPE__ or similar magic and then complains about unsigned long vs int etc. (even though all the same to the CPU).  The main thing is to make sure that %d %ld and %lld all "do the right thing" for 32, 32, and 64 bits (without warning when gcc is set to "max griping").